deploy/cdk/test/helpers.ts

```diff
@@ -1,3 +1,6 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
```

Comment:
Why disable on the whole file?

Reply:
There's going to be a lot of mocking in this file that require casting as `any`, so figured it would just be cleaner than adding it all over the place